app/src/androidTest/java/org/mozilla/fenix/ui/NavigationToolbarTest.kt

```diff
@@ -138,4 +141,46 @@ class NavigationToolbarTest {
             verifyFindNextInPageResult("1/1")
         }.closeFindInPage { }
     }
+
+    @Test
+    fun verifySecurePageSecuritySubMenuTest() {
+        val defaultWebPage = "https://mozilla-mobile.github.io/testapp/loginForm"
+        val defaultWebPageTitle = "Login_form"
+        val connectionSecurity = "Connection is secure"
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(defaultWebPage.toUri()) {
+        }.openSiteSecuritySheet {
+            verifyQuickActionSheet(defaultWebPage.substring(8, 32), connectionSecurity)
```

Comment:
This is stripping prefix from a URL? We can use URL and `authority()` or `path()` from a constructed URL object https://developer.android.com/reference/kotlin/java/net/URL#getauthority
https://codetinkering.com/how-to-remove-url-prefix-java/

Reply:
No, we need only the host.
Switched to `tryGetHostFromUrl()` and seems to be working fine.